website/src/components/HomepageFeatures/index.tsx

```
@@ -0,0 +1,71 @@
+import styles from './styles.module.css';
+
+type FeatureItem = {
+  title: string;
+  Svg: React.ComponentType<React.ComponentProps<'svg'>>;
+  description: ReactNode;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: 'Secure by Design',
+    Svg: require('@site/static/img/undraw_docusaurus_mountain.svg').default,
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)